<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(100, 100, 100)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCustomPlot" name="customPlot" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>500</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_max_x">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Nimbus Sans [urw]</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QDoubleSpinBox{
color: black;
border: 2px solid gray;
border-radius: 10px;
border-style: outset;
background: rgb(200, 200, 200);
padding: 5px;
}
QDoubleSpinBox::up-arrow {
background : gray;
}
QDoubleSpinBox::down-arrow {
background : gray;
}
</string>
         </property>
         <property name="prefix">
          <string>max x = </string>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_min_y">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Nimbus Sans [urw]</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QDoubleSpinBox{
color: black;
border: 2px solid gray;
border-radius: 10px;
border-style: outset;
background: rgb(200, 200, 200);
padding: 5px;
}
QDoubleSpinBox::up-arrow {
background : gray;
}
QDoubleSpinBox::down-arrow {
background : gray;
}
</string>
         </property>
         <property name="prefix">
          <string>min y = </string>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>-10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_max_y">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Nimbus Sans [urw]</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QDoubleSpinBox{
color: black;
border: 2px solid gray;
border-radius: 10px;
border-style: outset;
background: rgb(200, 200, 200);
padding: 5px;
}
QDoubleSpinBox::up-arrow {
background : gray;
}
QDoubleSpinBox::down-arrow {
background : gray;
}
</string>
         </property>
         <property name="prefix">
          <string>max y = </string>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_min_x">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Nimbus Sans [urw]</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QDoubleSpinBox{
color: black;
border: 2px solid gray;
border-radius: 10px;
border-style: outset;
background: rgb(200, 200, 200);
padding: 5px;
}
QDoubleSpinBox::up-arrow {
background : gray;
}
QDoubleSpinBox::down-arrow {
background : gray;
}
</string>
         </property>
         <property name="prefix">
          <string>min x = </string>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>-10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>155</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Nimbus Sans [urw]</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
color: black;
border: none;
border-radius: 10px;
border-style: outset;
background: rgb(200, 200, 200);
padding: 5px;
}

QPushButton:hover {
background: rgb(223, 223, 0);
}
</string>
         </property>
         <property name="text">
          <string>Tracing</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_step">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Nimbus Sans [urw]</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QDoubleSpinBox{
color: black;
border: 2px solid gray;
border-radius: 10px;
border-style: outset;
background: rgb(200, 200, 200);
padding: 5px;
}
QDoubleSpinBox::up-arrow {
background : gray;
}
QDoubleSpinBox::down-arrow {
background : gray;
}
</string>
         </property>
         <property name="prefix">
          <string>step = </string>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
